### PR TITLE
updating fp data TM-3711 

### PR DESCRIPTION
**Pull request by SidelineCory24**
**Need to run new migrate to update Frequent Positions table**
😺
Dual Merge: 
- [FE PR](https://github.com/MetaPhase-Consulting/State-TalentMAP/pull/2400)

To-do

- [x] update table with the correct/need frequent position data
- [x] fp data should show on the front end and work as expected on the FE (assuming FE ticket is working as expected)

### CODE REVIEW

**Comment by scott062** [comment]
Remove unused seed file

File: seeds/040_frequent_positions_seed.js [hunk 1]
```diff
@@ -1,19 +1,12 @@
...
-      const fp = [];
-      frequentpos.forEach(freqP => {
-        fp.push({
-          ...freqP,
-          posgradecode: _.sample(grades)
-        })
-      })
-      return knex('frequentpositions').insert(fp);
-    });
+      return knex.select('pos_seq_num as posseqnum').from('positions').limit(10)
```
Comment (SidelineCory24) [inline seeds/040_frequent_positions_seed.js:7]: setting the limit on what to show on the FE in the FP tab

**Comment by elizabeth-jimenez** [comment]
> > Remove unused seed file
> 
> done

You removed the reference to the seed file, but you can also go ahead and remove the whole seed file, since it shouldn't be used anywhere else

**Comment by SidelineCory24** [comment]
> > > Remove unused seed file
> > 
> > 
> > done
> 
> You removed the reference to the seed file, but you can also go ahead and remove the whole seed file, since it shouldn't be used anywhere else

I spoke too soon. 😅

File: migrations/081_frequentpositions.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+exports.down = function (knex) {
+  return knex.schema
+    .alterTable('frequentpositions', function (table) {
+      table.string('posorgshortdesc')
+      table.string('posnumtext')
+      table.string('posgradecode')
+      table.string('postitledesc')
+      table.dropForeign(posseqnum)
+    });
+};
```
Comment (pveland621) [inline migrations/081_frequentpositions.js:21]: Was wondering where the actual migration files were. How often are these migrations run? Only when the migrate command is called? And does it run every migration or just specific ones? In my previous job, all the migrations for a specific release were packaged together and were able to be run in that sort of modular capacity but it seems like there's a lot more rules on the databases/data sources for TM so they don't get so out of whack that you need to run migrations that often.

**Review by scott062 — APPROVED** [review]

File: migrations/081_frequentpositions.js (hunk 2 above)
Comment (SidelineCory24) [reply migrations/081_frequentpositions.js]: Question 1: the migrations files are here in this folder, with the filename `xxx_migrationfilename`
Question 2: correct, the migrations are only run when you run them with the docker-compose ... migrate command
Question 3: all of the migrations are ran at once, I tried running a specific migration file but it didn't like my syntax

@pveland621

**Pull request merged**